R/redcap_toform.R

```diff
@@ -0,0 +1,104 @@
+#' Convert manually downloaded REDCap data into a list of forms
+#'
+#' Similar to \code{redcap_export_byform}, this function tries to split a
+#' manually downloaded dataset into it's constituent forms. While use of the API
+#' allows individual forms to be downloaded, with a manual download, only the data
+#' dictionary is available as auxillary information.
+#'
+#' @param data imported REDCap data
+#' @param datadict data dictionary downloaded manually from REDCap
+#' @param metadata metadata downloaded from REDCap API
```

Comment:
should we maybe also provide the option to download this directly when providing token and url?